cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -917,36 +917,32 @@ SwiftGenerator::validateMetadata(const UnitPtr& u)
     u->visit(&visitor);
 }
 
-//
-// Get the fully-qualified name of the given definition. If a suffix is provided,
-// it is prepended to the definition's unqualified name. If the nameSuffix
-// is provided, it is appended to the container's name.
-//
-namespace
+string
+SwiftGenerator::getRelativeTypeString(const ContainedPtr& contained, const string& currentModule)
```

Comment:
Just to note, there's no risk of passing in a built-in type here, since `Builtin` isn't `Contained`.
So it couldn't be passed into this function.